### PR TITLE
SC-661 : Initial transition to open source repository.

### PR DESCRIPTION
**Pull request by SUB-BJ**

### CODE REVIEW

File: .idea/encodings.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (substratumservices) [inline .idea/encodings.xml]: add to .gitignore or simplify the .idea exclusions

File: .idea/vcs.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (substratumservices) [inline .idea/vcs.xml]: add to .gitignore or simplify the .idea exclusions

File: .idea/encodings.xml (hunk 1 above)
Comment (SUB-BJ) [reply .idea/encodings.xml]: fixed

File: .idea/vcs.xml (hunk 2 above)
Comment (SUB-BJ) [reply .idea/vcs.xml]: fixed

**Pull request merged**